webpack.config.js

```diff
@@ -26,27 +27,29 @@ function buildWithEnv(mode, outputFile) {
     devtool: "source-map",
     output: {
       path: __dirname + "/dist",
-      filename: outputFile
+      filename: outputFile,
     },
     resolve: {
       modules: [path.resolve("./node_modules"), path.resolve("./src")],
-      extensions: [".json", ".js"]
+      extensions: [".json", ".js"],
     },
     module: {
-      rules: []
+      rules: [],
+    },
+    plugins: [],
+    optimization: {
+      concatenateModules: false,
```

Comment:
This field was set solely to solve [this](https://github.com/webpack-contrib/closure-webpack-plugin/issues/90) problem, where `<SomeModule> has no id` kept being thrown as an error.